packages/shared/src/hooks/useRequestProtocol.ts

```diff
@@ -1,12 +1,12 @@
 import { useMemo } from 'react';
 import request from 'graphql-request';
-import { useQueryClient } from 'react-query';
+import { useQueryClient } from '@tanstack/react-query';
 import { RequestProtocol, REQUEST_PROTOCOL_KEY } from '../graphql/common';
 
 export const useRequestProtocol = (): RequestProtocol => {
   const client = useQueryClient();
   const { requestMethod, fetchMethod, isCompanion } =
-    client.getQueryData<RequestProtocol>(REQUEST_PROTOCOL_KEY) || {};
+    client.getQueryData<RequestProtocol>([REQUEST_PROTOCOL_KEY]) || {};
```

Comment:
I these kind of places I would just export the array from `REQUEST_PROTOCOL_KEY`

Reply:
It will also minimize amount of code changes probably?

Reply:
Yeah did it for other ones, just missed this one 😢